### DISCUSSION TITLE
Is this a bug?

### DISCUSSION THREAD

**Discussion by rustydanny** [valinet/ExplorerPatcher#2122] [Q&A]
My taskbar does not stay on top of applications when they're NOT maximized.
Doesn't matter what application it is: Excel, Chrome, Notepad, Windows Explorer

When the window's maximized, everything's as it should be, with the bottom of the window touching the top of the taskbar.

Here is my Windows/EP info:
This PC > Properties > Windows 11 Pro, OS Build: 22621.2283
Explorer Patcher says "Running on Windows 11, OS Build 10.0.22621.2283."
Explorer Patcher Version: 22621.1992.56.3

**Comment by Poopooracoocoo** [comment]
There's a new EP update by the way. I don't see this issue on my system.

**Comment by rustydanny** [comment]
Just installed the new update, restarted my laptop, and the issue's still there.

**Comment by pyrates999** [comment]
Please post explorer patcher version for the new update you just installed of it.

Please include what settings you changed in explorer patcher properties application.

Please take a screen shot for what you mean.
